time.c

```diff
@@ -48,7 +49,8 @@
 static ID id_submicro, id_nano_num, id_nano_den, id_offset, id_zone;
 static ID id_nanosecond, id_microsecond, id_millisecond, id_nsec, id_usec;
 static ID id_local_to_utc, id_utc_to_local, id_find_timezone;
-static ID id_year, id_mon, id_mday, id_hour, id_min, id_sec, id_isdst;
+static ID id_year, id_mon, id_mday, id_hour, id_min, id_sec, id_subsec, id_isdst, id_dst;
+static ID id_month, id_day, id_yday, id_wday;
```

Comment:
Seems odd to add static IDs for values never used as IDs and only used as symbols.  I recommend using `static VALUE sym_*` for all of the symbols this uses.

Reply:
That was done for homogenity. Most of necessary IDs are already defined in the file, so I needed to add only a few, and it was either
1. this (add `ID`s, and use `ID2SYM`), or
2. add symbol representation for those where `ID`s where already present, or
3. break homogeneity (use `ID2SYM` for existing `ID`s, and just symbols for those I added)

Which looks like the most reasonable option to you?..

Reply:
I think option 2 is what I recommended, and that sounds like the most reasonable option.